Exiled.API/Features/Player.cs

```diff
@@ -687,12 +687,19 @@ public Item CurrentItem
 
             set
             {
-                if (!Inventory.UserInventory.Items.TryGetValue(value.Serial, out _))
+                if (value == null || value.Type == ItemType.None)
                 {
-                    AddItem(value.Base);
+                    Timing.CallDelayed(0.5f, () => Inventory.ServerSelectItem(0));
```

Comment:
Why is this delayed?

Reply:
It's not needed in this case, since it's an empty item, the delay before AddItem() normally is because the item is not synced to the client until the frame after AddItem() is called, so there's a few frames of delay to make sure their client actually sees the item.

Reply:
I have removed the delay.